README.md

```diff
@@ -89,61 +89,143 @@ builds like `gcc` generate a *lot* of output), and builds timing out
 
 ## Testing conda builds locally
 
-Given a fairly empty *disposable* Ubuntu x86-64 test environment (eg,
-created with Docker, or Vagrant), it is possible to simulate *part* of
-what Travis CI will do to test building individual toolchain architectures
-locally.
+It is recommend to build these packages in a fresh disposable environment
+such as a clean docker container.
+While the goal is for the conda environment to be totally self contained,
+there is a constant battle to make sure this happens.
 
-This can be done with something like:
+The commands from the following subsections are enough to build a package
+in a clean docker container based on the *ubuntu* Docker image.
```

Comment:
Is "any" version of Ubuntu supported? Or is is bionic only, for now?

I suggest using "*container image*" instead of "*Docker image*".

Reply:
Changed to "The commands from the following subsections were tested to be enough to build a package in a clean container based on `ubuntu` (`16.04`-`20.04`) or `debian` (`8`-`10`) Docker image." after I tested them on these OSs.